swagger/transforms.go

```diff
@@ -0,0 +1,159 @@
+package swagger
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/go-openapi/jsonreference"
+	"github.com/go-openapi/spec"
+)
+
+// TransformErrors transforms the errors of the swagger spec object. This means
+// both verifying that the required errors exist, and also adding the 400 / 500
+// responses to any operation that doesn't have them defined.
+func TransformErrors(s spec.Swagger) error {
```

Comment:
the `spec.Swagger` is get transformed because we're adding the default 400 + 500 responses. That said, I agree that it doesn't describe the function very well. maybe i should name it `ValidateErrors` since that's much more of what it does then add the extra responses.